### ISSUE TITLE
Champs du formulaire non intégrés dans le ticket

### ISSUE THREAD

**Issue by JeanBart59** [InfotelGLPI/metademands#136]
Bonjour,
nous rencontrons une difficulté avec la transformation en ticket de certains formulaires.
En effet, certains champs ne sont pas repris dans le ticket. Les informations sont pourtant bien enregistrées (via "voir le formulaire").
Supprimer le champ du formulaire pour le recréer un champ identique peut résoudre le problème mais pas toujours, et parfois cela fait "disparaître" un autre champ.
Il semble que les champs "boutons radio" soient plus affectées, mais d'autres types de champ peuvent être concernés.
Le problème est apparu depuis la version 3.3.10
Merci

**Comment by avallano2817** [comment]
Bonjour, 

Constat un peu identique suite à la mise à jour en 3.3.10 des champs cachés liste déroulantes qui doivent apparaitre à la sélection d'un bouton radio ne s'affiche que dans l'aperçu, il ne s'affiche plus pour les utilisateurs.

**Comment by tsmr** [comment]
Merci de vérifier dans Outils sur la page metademands/front/config.form.php afin de voir si des données de formulaires sont à migrer / corriger

**Comment by JeanBart59** [comment]
Bonjour,
j'ai toute une liste de champs dans "Valeurs personnalisées vides", à quoi cela correspond-t-il?

**Comment by tsmr** [comment]
il faut cliquer sur l'icone de droite (pour les migrer vers des valeurs différente de 0)

**Comment by JeanBart59** [comment]
Oui, effectivement, la purge de la liste a résolu le problème. Est-ce lié au passage à la nouvelle version?
Au moment de l'enregistrement du formulaire, j'ai un double message d'erreur : "Error :  An error has occured, but the trace of this error could not recorded because of a problem accessing the log file"

**Comment by avallano2817** [comment]
Bonjour, 
Pour moi cela ne corrige en rien le problème.
Les listes ne veulent toujours pas réapparaitre.

**Comment by tsmr** [comment]
@avallano2817 pourriez-vous envoyer le fichier XML ?

**Comment by tsmr** [comment]
@JeanBart59 c'est problème d'accès aux fichiers du coeur cela surement

**Comment by tsmr** [comment]
@avallano2817 vous parlez du champ Société ? Je ne vois pas d'options dans le fichier XML pour ce champ

**Comment by avallano2817** [comment]
Je viens de refaire un export, je me suis peut être trompé a force de test et tentative de rétablissement. 
Dans la version (3), j'ai bien mes services qui apparaissent en mode "Aperçu" mais des que je lance la création d'un formulaire depuis un profil "technicien", il n'y a que le premier qui apparait.

**Comment by avallano2817** [comment]
Je viens de refaire un export, je me suis peut être trompé a force de test et tentative de rétablissement. 
Dans la version (3), j'ai bien mes services qui apparaissent en mode Apercu mais des que je lance la création d'un formulaire depuis un profil technicien, il n'y a que le premier qui apparait.

**Comment by tsmr** [comment]
Please test last version. I close
